.gitignore

```diff
@@ -17,4 +17,4 @@
 /tests/TextUI/*.out
 /tests/TextUI/*.php
 /vendor
-
+/.phpunit.result.cache
```

Comment:
This is not needed if we do not enable result caching for PHPUnit's own test suite.

Reply:
True! However, I can also imagine a user being surprised by a change in `git status` when they turn on the feature and all of a sudden an untracked file appears.

Reply:
I would also keep it, I suggest to always have `.php_cs.cache` in .gitignore even if `.php_cs.dist` is not having cache enabled (but `.php_cs` can have, right ?)